README.md

```
@@ -189,6 +189,25 @@ releases! 🌟
+> ```bash
+> # First run, or after editing source: build first, then up.
+> $ docker compose -f docker-compose.yml build ragflow-cpu
+> $ docker compose -f docker-compose.yml up -d ragflow-cpu
+>
+> # Skip build and pull the prebuilt image instead (overrides build):
+> $ docker compose -f docker-compose.yml pull ragflow-cpu
+> $ docker compose -f docker-compose.yml up -d ragflow-cpu
+> ```
+>
+> The build runs `npm run build` inside `Dockerfile`, so the resulting image always contains the frontend built from your local `web/src`.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)